simtools/simtel/simtel_runner.py

```diff
@@ -168,7 +168,7 @@ def run(self, test=False, force=False, input_file=None, run_number=None):
             self._logger.error(msg)
             raise RuntimeError(msg)
 
-        if not self._shall_run(run_number=run_number) and not force:
+        if not self._shall_run() and not force:
```

Comment:
And you are sure that the run number is not needed? This seems to be a functional change.

Reply:
Don't really understand what the purpose of shall_run is here: https://github.com/gammasim/simtools/blob/b010eee27884d85353cac010f0ae17e19ed291a0/simtools/simtel/simtel_runner.py#L231
It only returns False. Do you know?

Reply:
I think we need this function in case this is not re-implemented in the child classes (as e.g., in simulated_light_emission).

Reply:
Ok this seems indeed needed. The `run_number` however is not required to be passed.